src/main/java/io/jenkins/plugins/audit/CustomHTMLLayout.java

```diff
@@ -0,0 +1,412 @@
+        final StringWriter sw = new StringWriter();
+        final PrintWriter pw = new PrintWriter(sw);
+        try {
+            throwable.printStackTrace(pw);
+        } catch (final RuntimeException ex) {
+            // Ignore the exception.
+        }
+        pw.flush();
+        final LineNumberReader reader = new LineNumberReader(new StringReader(sw.toString()));
+        final ArrayList<String> lines = new ArrayList<>();
+        try {
+          String line = reader.readLine();
+          while (line != null) {
+            lines.add(line);
+            line = reader.readLine();
+          }
+        } catch (final IOException ex) {
+            if (ex instanceof InterruptedIOException) {
+                Thread.currentThread().interrupt();
+            }
+            lines.add(ex.toString());
+        }
+        boolean first = true;
+        for (final String line : lines) {
+            if (!first) {
+                sbuf.append(TRACE_PREFIX);
+            } else {
+                first = false;
+            }
+            sbuf.append(Transform.escapeHtmlTags(line));
+            sbuf.append(Strings.LINE_SEPARATOR);
+        }
+    }
+
+    private StringBuilder appendLs(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s).append(Strings.LINE_SEPARATOR);
+        return sbuilder;
+    }
+
+    private StringBuilder append(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s);
+        return sbuilder;
+    }
+
+    /**
+     * Returns appropriate HTML headers.
+     * @return The header as a byte array.
+     */
+    @Override
+    public byte[] getHeader() {
+        final StringBuilder sbuf = new StringBuilder();
+        append(sbuf, "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" ");
+        appendLs(sbuf, "\"http://www.w3.org/TR/html4/loose.dtd\">");
+        appendLs(sbuf, "<html>");
+        appendLs(sbuf, "<head>");
+        append(sbuf, "<meta charset=\"");
+        append(sbuf, getCharset().toString());
+        appendLs(sbuf, "\"/>");
+        append(sbuf, "<title>").append(title);
+        appendLs(sbuf, "</title>");
+	appendLs(sbuf, "<style type=\"text/css\">");
+        appendLs(sbuf, "<!--");
+        append(sbuf, "body, table {font-family:").append(font).append("; font-size: ");
+        appendLs(sbuf, headerSize).append(";}");
+        appendLs(sbuf, "th {background: #f0f0f0; font-weight: bold; text-align: left;}");
+	appendLs(sbuf, "tr:nth-child(odd) {background-color: #fbfbfb;}");
+	appendLs(sbuf, "tr:hover {background-color: #e8e8e8 !important;}");
+        appendLs(sbuf, "-->");
+        appendLs(sbuf, "</style>");        
+	appendLs(sbuf, "</head>");
+        appendLs(sbuf, "<body bgcolor=\"#FFFFFF\" topmargin=\"6\" leftmargin=\"6\">");
+        appendLs(sbuf, "<hr size=\"1\" noshade=\"noshade\">");
+        appendLs(sbuf, "Log session start time " + new java.util.Date() + "<br>");
```

Comment:
This might be a little nicer done using `DateTimeHelper.formatDateISO(System.currentTimeMillis())` which would print out an ISO-8601 date.

Reply:
I couldn't find the package for DateTimeHelper class. I've replaced it with different class with same output formatting.

Reply:
https://github.com/jenkinsci/audit-log-plugin/blob/master/src/main/java/io/jenkins/plugins/audit/helpers/DateTimeHelper.java

Reply:
@jvz gotcha.